dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
If I understand your [reply](https://github.com/qbittorrent/qBittorrent/pull/6806#issuecomment-306853939) correctly, qbt is not ready for multi-user usage, then this should be `PrivateTmp=false` for now.